diff -aur sonic-pi-2.11.1.orig/app/gui/qt/rp-build-app sonic-pi-2.11.1/app/gui/qt/rp-build-app
--- sonic-pi-2.11.1.orig/app/gui/qt/rp-build-app	2017-01-15 16:34:46.837045750 +0100
+++ sonic-pi-2.11.1/app/gui/qt/rp-build-app	2017-01-15 16:35:15.847235827 +0100
@@ -1,8 +1,5 @@
 #!/bin/bash
 
-echo "We're working to update the build process for SP 2.11, this script hasn't been updated yet, and may not work, try ./build-ubuntu-app instead!"
-exit
-
 set -eux
 
 cd $(dirname $0)
