--- ./deheader	(original)
+++ ./deheader	(refactored)
@@ -32,7 +32,7 @@
 """
 # SPDX-License-Identifier: BSD-2-Clause
 
-from __future__ import print_function
+
 
 import sys, os, getopt, time, re, operator, subprocess
 
@@ -41,7 +41,7 @@
 try:
     from subprocess import getstatusoutput
 except ImportError:
-    from commands import getstatusoutput
+    from subprocess import getstatusoutput
 
 BATON_DEBUG    = 1
 PROGRESS_DEBUG = 2
