diff -Naur sonic-pi-2.7.0/app/gui/qt/SonicPi.pro sonic-pi-2.7.0.new/app/gui/qt/SonicPi.pro
--- sonic-pi-2.7.0/app/gui/qt/SonicPi.pro	2015-09-10 15:29:43.000000000 +0200
+++ sonic-pi-2.7.0.new/app/gui/qt/SonicPi.pro	2015-09-14 19:24:03.890260279 +0200
@@ -17,13 +17,6 @@
 #
 #-------------------------------------------------
 
-# -- Change to match the location of QScintilla on your system
-#
- LIBS += -L/Users/sam/Downloads/tmp/QScintilla-gpl-2.9/Qt4Qt5 -lqscintilla2
- INCLUDEPATH += /Users/sam/Downloads/tmp/QScintilla-gpl-2.9/Qt4Qt5
- DEPENDPATH += /Users/sam/Downloads/tmp/QScintilla-gpl-2.9/Qt4Qt5
-# --
-
 QT       += core gui concurrent network
 
 greaterThan(QT_MAJOR_VERSION, 4): QT += widgets
@@ -102,7 +95,7 @@
 RC_FILE = SonicPi.rc
 
 ICON = images/app.icns
-LIBS         += -lqscintilla2
+LIBS         += -lqt5scintilla2
 
 win32 {
 	install_qsci.files = $$[QT_INSTALL_LIBS]\qscintilla2.dll
